src/stories/Library/availability-label/AvailabilityLabel.tsx

```diff
@@ -2,8 +2,11 @@ import { AvailabilityLabelPropsType } from "../../availability-label/types";
 import { Pagefold } from "../pagefold/Pagefold";
 import { withAvailabilityProps } from "./abilityLabel.hoc";
 
-export const AvailabilityLabel = (props: AvailabilityLabelPropsType) => {
-  const { manifestation, availability, status } = props;
+export const AvailabilityLabel: React.FC<AvailabilityLabelPropsType> = ({
```

Comment:
Consider if something is missing here. It does not seem as if we import `React.FC`. 

I wonder why this change is needed anyway. The version before was shorter 🤔 .

Reply:
Importing react is no longer necessary since React 17:
![image](https://user-images.githubusercontent.com/28546954/190188059-2d381868-89e4-46a2-a436-f5583596868c.png)

And I just changed it for consistency's sake, as that's how we implement our components throughout the project. Note that there might be some leftover older components from before our time where this is not the case, but generally, this way is what I'm used to seeing. :)